module/input/NatNet/src/NatNet.cpp

```diff
@@ -201,6 +200,8 @@ namespace module::input {
                 // Stop processing
                 return;
             }
+
+            // Should probably be saving the informaion of the frame here
```

Comment:
what does this mean? it looks like frame information is being saved into a message above?

Reply:
This is leftover from an in progress build, I'll remove it